controllers/storagecluster/storageclasses.go

```diff
@@ -366,6 +371,10 @@ func newNonResilientCephBlockPoolStorageClassConfiguration(initData *ocsv1.Stora
 		},
 		isClusterExternal: initData.Spec.ExternalStorage.Enable,
 	}
+	if initData.GetAnnotations()[keyRotationEnableAnnotation] == "false" {
+		util.AddAnnotation(scc.storageClass, keyRotationEnableAnnotation, "false")
```

Comment:
if the annotation is removed on the storagecluster same need to be removed on the SC as well right?

Reply:
The StorageClass that we return from this function replaces the existing one. And we only add the annotation to this NEW StorageClass if we find it on StorageCluster.

If the annotation is removed, the new SC won't have it present and since the annotations will be different between this NEW SC and existing SC, the existing SC (with disable annotation) will be replaced with this new SC (which does not have the annotation set)